drivers/flash/flash_nop_device/flash_nop_device.c

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#define DT_DRV_COMPAT zephyr_sim_flash
+
+#include <device.h>
+#include <drivers/flash.h>
+#include <init.h>
+#include <kernel.h>
+#include <sys/util.h>
+#include <random/rand32.h>
```

Comment:
Do we need to include random, stats and string ?

Reply:
strings is bringing memset, others have been removed.